### PR TITLE
Keep original path with `appName` of `RnDiffApp` to View or Download files

### PR DESCRIPTION
**Pull request by benomatis**
If you change the App Name on top, then click on View file or the download icon, you'll get redirected to a dead link because the app name is also replaced in the path to the one the user set.

I changed it so it reverts to the default RnDiffApp only for view and download to prevent the 404 error.

### CODE REVIEW

**Comment by pvinis** [comment]
Good catch. I think we should do some work there for actually supporting this better, but this is not for this PR.

**Comment by lucasbento** [comment]
> @pvinis: Good catch. I think we should do some work there for actually supporting this better, but this is not for this PR.

Would be nice to create an issue for this if you already have ideas in mind 😄

**Comment by pvinis** [comment]
Let's merge this for now, because it's better to show the file than to show an error 😅

**Comment by benomatis** [comment]
> > @pvinis: Good catch. I think we should do some work there for actually supporting this better, but this is not for this PR.
> 
> Would be nice to create an issue for this if you already have ideas in mind 😄

I guess this will have to be done with apache mod_rewrite, to keep the URL with the custom app name in the path, but show the content of the file with the original path.

**Comment by pvinis** [comment]
My idea was to not show the original, basically #208.

File: src/utils.js [hunk 1]
```diff
@@ -15,6 +15,9 @@ export const getBinaryFileURL = ({ version, path }) =>
 export const removeAppPathPrefix = (path, appName) =>
   path.replace(new RegExp(`${appName || 'RnDiffApp'}/`), '')
 
+export const getOriginalPath = (path, appName) =>
+  path.replace(new RegExp(`${appName}`), 'RnDiffApp')
```
Comment (benomatis) [inline src/utils.js]: @pvinis Following this change, for some reason, ios paths show the app name twice at the beginning resulting in a 404 again.

eg https://github.com/react-native-community/rn-diff-purge/raw/release/0.59.1/RnDiffAppRnDiffApp/package.json

notice `RnDiffAppRnDiffApp` in the path just after the version number.

I'm wondering it it has something to do with my implementation of the `replace` method.

Should it maybe just be this...?

```suggestion
  path.replace(appName, 'RnDiffApp')
```

Comment (benomatis) [reply src/utils.js]: I tried this, it's not that... I'll try to fix this and open a new pull request, if you have any idea, I'd welcome any pointers ;) Thanks.

Comment (benomatis) [reply src/utils.js]: in fact, this seems to be a problem with files that have the app name in it (like some of the ios ones)...

Comment (benomatis) [reply src/utils.js]: Should be fixed in https://github.com/react-native-community/upgrade-helper/pull/209

**Comment by elie222** [comment]
Possibly broke this:
https://react-native-community.github.io/upgrade-helper/?from=0.60.6&to=0.61.5

**Comment by benomatis** [comment]
> Possibly broke this:
> https://react-native-community.github.io/upgrade-helper/?from=0.60.6&to=0.61.5

Looks like it's broken by eeea806ed25e9e1cbca66f5c2f1dd0f1d907163a instead.

**Comment by lucasbento** [comment]
Thanks for notifying this issue @elie222 and thank you @benomatis for pinpoint where the problem was.

I just reverted the version bump in https://github.com/react-native-community/upgrade-helper/commit/eeea806ed25e9e1cbca66f5c2f1dd0f1d907163a so the latest build should be working.

**Comment by benomatis** [comment]
@lucasbento Yup, I can confirm, it works with the old version.
@elie222 All good for you too?

**Comment by elie222** [comment]
Yup, thanks!
https://react-native-community.github.io/upgrade-helper/?from=0.60.6&to=0.61.5 works great

**Pull request merged**